instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integrationtests/utils/AbstractProfilingRule.kt

```diff
@@ -24,7 +24,8 @@ internal abstract class AbstractProfilingRule<T> : TestRule {
     abstract fun measureAfterAction(): T
     abstract fun compareWithThreshold(before: T, after: T, threshold: T)
 
-    fun profile(action: () -> Unit, threshold: T) {
+    fun profile(action: () -> Unit, threshold: T, warmupAction: () -> Unit = {}) {
```

Comment:
that's nice, but nitpicking a bit, I'd put the warmup action before the action in the list of parameter. Seems more logical ?